testplan/web_ui/testing/src/AssertionPane/AssertionTypes/DictAssertions/dictAssertionUtils.js

```diff
@@ -1,3 +1,4 @@
+import _uniq from 'lodash/uniq';
 import {any, sorted, domToString} from './../../../Common/utils';
```

Comment:
you'll see I change `Set` to just a plain array in a few places. The report is being stored in the app's state, and redux doesn't handle `Set`.